src/actions/like.actions.js

```diff
@@ -1,30 +1,25 @@
-import { toast } from 'react-toastify';
 import exceptionHandler from '../utils/exceptionHandler';
 import actionTypes from '../constants/article.constants';
 import http from '../utils/httpService';
 
-export const likeSuccess = (articleId, article) => ({
+export const likeSuccess = likeCount => ({
   type: actionTypes.LIKE_SUCCESS,
-  articleId,
-  article,
+  likeCount,
 });
 
-export const likeFailure = article => ({
+export const likeFailure = () => ({
   type: actionTypes.LIKE_FAILURE,
-  article,
 });
 
-export const likeArticle = (articleId, article) => {
+export const likeArticle = articleId => {
   return async dispatch => {
     try {
-      dispatch(likeSuccess(articleId, article));
       const { data: result } = await http.post(`/likes/${articleId}`);
```

Comment:
You don't need to rename this @Akinmyde

Reply:
The implementation has change, that was why i changed it